osquery/tables/system/darwin/sysctl_utils.cpp

```diff
@@ -110,6 +167,11 @@ void genControlInfo(int* oid,
     }
   }
 
+  // If this MIB was set using sysctl.conf add the value.
+  if (config.count(r.at("name")) > 0) {
+    r["config_value"] = config.at(r["name"]);
+  }
```

Comment:
There are to look ups in the map. Using `.find()` you could do it with only one.
```c++
auto const cfgValueIt = config.find(r.at("name"));
if (cfgValueIt != config.end()) {
  r["config_value"] = cfgValueIt->second;
}
```